content/4_Gold/LIS.mdx

```diff
@@ -1,9 +1,8 @@
 ---
 id: lis
 title: 'Longest Increasing Subsequence'
-author:
-  Michael Cao, Benjamin Qi, Andi Qu, Andrew Wang, Dong Liu, Siyong Huang,
-  Aryansh Shrivastava
+author: Michael Cao, Benjamin Qi, Andrew Wang
+contributors: Andi Qu, Dong Liu, Siyong Huang, Aryansh Shrivastava
```

Comment:
Also, there's a "(By Aryansh Shrivastava)" lower down on this page, which we should probably remove